src/System.Management.Automation/engine/hostifaces/History.cs

```diff
@@ -991,24 +986,7 @@ public class InvokeHistoryCommand : PSCmdlet
         /// that is to be re-executed.
         /// </summary>
         [Parameter(Position = 0, ValueFromPipelineByPropertyName = true)]
-        public string Id
-        {
-            get
-            {
-                return _id;
-            }
-
-            set
-            {
-                if (_id != null)
-                {
-                    // Id has been set already.
-                    _multipleIdProvided = true;
-                }
-
-                _id = value;
-            }
-        }
+        public string[] Id { get; set; }
```

Comment:
Should we have an issue to update SDK?

Reply:
@iSazonov, not sure what the process is for that.

Reply:
The quest is more for MSFT team. /cc @daxian-dbw